gradle.properties

```diff
@@ -30,4 +30,6 @@ POM_LICENCE_NAME=The Apache Software License, Version 2.0
 POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0
 POM_LICENCE_DIST=repo
 POM_DEVELOPER_ID=yalantis
-POM_DEVELOPER_NAME=Yalantis
\ No newline at end of file
+POM_DEVELOPER_NAME=Yalantis
+
+android.useAndroidX=true
```

Comment:
No need to specify `android.enableJetifier=true` because this library and sample project do not use any third-party libraries with dependencies that need converting.